Week4/WEEK-4 SOLUTION/week-4 solution.ipynb

```diff
@@ -0,0 +1,407 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b2054568",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sqlite3 as sq\n",
+    "import datetime"
```

Comment:
Explain import datetime ?

Reply:
Here, we have imported the datetime module using the import datetime statement.